lib/offline_sort/chunk/input_output/message_pack.rb

```diff
@@ -25,6 +26,10 @@ def flush
           packer.flush
           super
         end
+
+        def open
```

Comment:
Should this change to define `open` be in ` OfflineSort::Chunk::InputOutput::Base`? It looks like in the base class it's already keeping a reference to `@io` so it should be fine to just move it there so that all the different chunk IOs can be re-opened?